code/modules/mob/living/carbon/xenomorph/Castes/Crusher.dm

```diff
@@ -313,6 +313,11 @@
 				X.stop_momentum(X.charge_dir)
 				return FALSE
 
+
+/obj/effect/forcefield/charge_act(mob/living/carbon/Xenomorph/X)
```

Comment:
There's quite a few conflicting methods here, and while the fog is not `unacidable`, there's a specific check for `istype(I, /obj/effect)` in the acid proc.
Crushers can't crush unacidable objects.
Thus, we should simply remove that check and make all effects unacidable. No need for this proc redefinition then.